<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositCalc</class>
 <widget class="QWidget" name="DepositCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>473</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>435</width>
    <height>473</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>435</width>
    <height>473</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="sum_line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма вклада</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="term_line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>123</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок размещения</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="start_line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Начало срока</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="procent_line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>123</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="term_line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Периодичность выплат</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="capital_proc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Капитализация процентов</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nalog_line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Налог</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="procent_line_calc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Начисленные проценты</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nalog_line_calc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Налог</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="sum_line_calc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма на вкладе к концу срока</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="calc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>421</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Расчитать</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sum">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> rub</string>
   </property>
   <property name="maximum">
    <double>9999999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="procent">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> %</string>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>999999999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="nalog">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> %</string>
   </property>
   <property name="maximum">
    <double>99999999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="capital_box">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="term">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QComboBox" name="year_month">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>55</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Лет</string>
   </property>
   <item>
    <property name="text">
     <string>Дней</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Месяцев</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Лет</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="payments">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>176</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ежедневно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Еженедльно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ежемесячно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ежегодно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>В конце срока</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="date">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>101</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="procent_calc">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nalog_calc">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="sum_calc">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>440</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="replenishment_list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Список пополнений</string>
   </property>
  </widget>
  <widget class="QPushButton" name="list_withdrawals">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Список снятий</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
